### ISSUE TITLE
Can a GDExtension compatibility layer be provided to better support Godot?

### ISSUE THREAD

**Issue by scgm0** [notgiven688/jitterphysics2#175]
I was pleasantly surprised that this library has a Godot demo (actually two!). So is it possible to add better support for Godot?

Godot allows the addition of a third-party physics engine via GDExtension: https://docs.godotengine.org/en/stable/classes/class_physicsserver3dextension.html

godot-dotnet is an experimental .NET binding for the GDExtension API that allows c# compilation to GDExtension: https://github.com/raulsntos/godot-dotnet

I've been trying it myself for hours, but I underestimated the difficulty of adding a physics engine to Godot, I didn't know much about Godot physics and jitterphysics 2, so I had to give up...
So this is more of a whim on my part (e.g. can we compare three physics engines written in c#, cpp, rust in Godot?). ), instead of a serious request, you can close the issue at will.

**Comment by notgiven688** [comment]
Good point. Unfortunately I do not have enough time to implement and support such a tool. Godot is a very good engine and will only get better. However there are no plans from my side to focus on a special implementation for any engine. I will leave this Issue open and add a "help wanted" sticker. Maybe someone wants to have a look into it.

**Comment by youfch** [comment]
https://github.com/[youfch/jitterphysics2_for_godot](https://github.com/youfch/jitterphysics2_for_godot)
In development, it can be used now

**Comment by youfch** [comment]
> [https://github.com/[youfch/jitterphysics2_for_godot](https://github.com/youfch/jitterphysics2_for_godot)](https://github.com/%5Byoufch/jitterphysics2_for_godot%5D(https://github.com/youfch/jitterphysics2_for_godot)) In development, it can be used now

Afterwards, it will be made public

**Comment by notgiven688** [comment]
@youfch Looking forward to your solution.
